packages/cashier/src/components/no-balance/__tests__/no-balance.spec.tsx

```diff
@@ -4,21 +4,32 @@ import { createBrowserHistory } from 'history';
 import { Router } from 'react-router';
 import { routes } from '@deriv/shared';
 import NoBalance from '../no-balance';
-
-jest.mock('Stores/connect', () => ({
-    __esModule: true,
-    default: 'mockedDefaultExport',
-    connect: () => Component => Component,
-}));
+import { StoreProvider } from '../../../hooks';
 
 describe('<NoBalance />', () => {
     const history = createBrowserHistory();
+    let mockRootStore;
+
+    beforeEach(() => {
+        mockRootStore = {
+            client: { currency: 'USD' },
+            modules: {
+                cashier: {
+                    deposit: { is_deposit_locked: false },
+                    general_store: { setCashierTabIndex: jest.fn() },
+                },
+            },
+        };
+    });
 
     it('component should render', () => {
         render(
             <Router history={history}>
-                <NoBalance is_deposit_locked={false} />
-            </Router>
+                <NoBalance />
```

Comment:
Suggestion: As this render argument is identical for all test cases, can it be a function used in test cases?

Reply:
@hamid-deriv I think in this way we'll lose the readability of the code, so let it stay like this